website/docs/attached-class.md

```diff
@@ -0,0 +1,53 @@
+---
+id: attached-class
+title: T.attached_class
+---
+
+`T.attached_class` can be used to refer to the type of instances from a
+singleton-context, in a way that is friendly to inheritance. Consider the
+following example:
```

Comment:
Suggestion: the first code block people should see on the page is proper usage.

Before motivating the use case, you might want to add a quick snippet for people who know what they're doing and just need to remember the syntax.

Reply:
I like that, I'll put together something small that motivates it, but is distinct from the other cases before merging.